src/platform/input/mouse-event.js

```diff
@@ -50,7 +50,7 @@ class MouseEvent {
              * @type {number}
              */
             this.y = coords.y;
-        } else if (isMousePointerLocked()) {
+        } else if (isMousePointerLocked() || event.type === 'mouseout') {
```

Comment:
why is this necessary?

Reply:
On the same note, what values are coords.x/y on a mouse out event?

Reply:
> why is this necessary?

`coords = mouse._getTargetCoords(event);` returns null and then the constructor just ends here:

https://github.com/playcanvas/engine/blob/7c582ab58e69ae21291771b586910a9e52bf4b62/src/platform/input/mouse-event.js#L56-L58

Resulting in the entire event being "uninitialized".

Reply:
Ah, I see 🤔 This seems fine to me